src/tests/ftest/io/ior_large.py

```diff
@@ -24,8 +24,9 @@
 from ior_test_base import IorTestBase
 
 
-class IorEightServers(IorTestBase):
-    """Test class Description: Runs IOR with 8 servers.
+class IorLarge(IorTestBase):
```

Comment:
(pylint-too-many-ancestors) Too many ancestors (8/7)

Reply:
You could remove this pylint error by adding this tag "# pylint: disable=too-many-ancestors" next to the class.

Reply:
I think this one should be taken care by our pylint file for all tests if such situation is there.